src/components/POLine/POLineForm.js

```diff
@@ -211,6 +212,7 @@ class POLineForm extends Component {
     const funds = getFundsForSelect(parentResources);
     const materialTypes = getMaterialTypesForSelect(parentResources);
     const identifierTypes = getIdentifierTypesForSelect(parentResources);
+    const contributors = getContributorsForSelect(parentResources);
```

Comment:
IMO it's better to rename `contributors` to `contributorsTypes` or `contributorsNameTypes`